.github/workflows/push-deploy-website.yml

```
@@ -51,5 +54,4 @@
           git config --global user.email "actions@github.com"
           git config --global user.name "gh-actions"
           yarn install --frozen-lockfile
-          yarn deploy
-
+          yarn deploy
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)